bridges/evm/contracts/IBridge.sol

```diff
@@ -0,0 +1,16 @@
+pragma solidity 0.5.14;
+pragma experimental ABIEncoderV2;
+
+interface IBridge {
+  /// Helper struct to help the function caller to decode oracle data.
+  struct VerifyOracleDataResult {
+    bytes data;
+    bytes32 codeHash;
+    bytes params;
+  }
+
+  /// @param _data The encoded data for oracle state relay and data verification.
```

Comment:
Let's just copy the whole comments from Bridge.sol
```
  /// Performs oracle state relay and oracle data verification in one go. The caller submits
  /// the encoded proof and receives back the decoded data, ready to be validated and used.
  /// @param data The encoded data for oracle state relay and data verification.
```

Reply:
done

Reply:
done